requirements.txt

```diff
@@ -66,7 +69,7 @@ packaging==23.1
     #   astropy
     #   matplotlib
 pandas==2.0.0
```

Comment:
I wonder if some of these dependencies of `skyreader` could be made optional so that they don't get dragged into the skymap-scanner. For example `pandas` and `meander` are not used though `matplotlib` itself is a dependency of `healpy` so I guess that cannot be separated.

Reply:
I agree.

I guess this pattern is a consequence of `skyreader` providing both the data exchange format _and_
a library to analyze the results. One could think of a scenario where both `skyreader` and `skymap_scanner` depend on a third package that only defines the data format, but probably that would be excessive fragmentation.

Reply:
That's right, it's a consequence of `skyreader`'s dependencies. I can make an install-extra for `skyreader` -- call it `plots`.

Reply:
`pandas` is used by `serialzie()`. I looked into this further and came to the conclusion that it's not worth it: https://github.com/icecube/skyreader/pull/5#issuecomment-1515035778